src/main/java/springsamurais/toyswapbackend/model/Listing.java

```diff
@@ -21,16 +23,10 @@ public class Listing {
     private Long id;
 
     @NotNull
+    @Column(unique = true)
```

Comment:
It might not matter that a "title" is unique (two users could have the exact same title for some reason and we wouldn't want to reject it if they accidentally both wrote "Awesome toy!") 

But this is a massive edge-case and not important now.